README.md

```diff
@@ -48,8 +50,10 @@ In `~/.aws/credentials` add the following:
 
 ```
 [membership]
-aws_access_key_id=[YOUR_AWS_ACCESS_KEY]
-aws_secret_access_key=[YOUR_AWS_SECRET_ACCESS_KEY]
+aws_access_key_id = [YOUR_AWS_ACCESS_KEY]
+aws_secret_access_key = [YOUR_AWS_SECRET_ACCESS_KEY]
+region = eu-west-1
```

Comment:
Did you end up needing this? This should be inherited from the default settings below in `~/.aws/config`. Interesting if you did, though.